routing/car_router.cpp

```diff
@@ -281,24 +281,28 @@ bool CarRouter::FindRouteMSMT(TFeatureGraphNodeVec const & sources,
           FindSingleRouteDispatcher(sourceEdge, targetEdge, delegate, mapping, route);
       switch (code)
       {
-      case NoError: return true;
-      case Cancelled: return false;
+      case StartPointNotFound:
+      case EndPointNotFound:
+      case RouteNotFound:
+        continue;
+
+      case NoError:
+      case Cancelled:
+      case InconsistentMWMandRoute:
+      case RouteFileNotExist:
+      case PointsInDifferentMWM:
```

Comment:
I guess in that case we want to write an log error message. It's a method for finding single shortest path. I'm adding a LOG(LERROR, ... ) in that case.